modules/cli/src/main/scala/scala/cli/commands/shared/HelpGroups.scala

```diff
@@ -0,0 +1,13 @@
+package scala.cli.commands.shared
+
+enum HelpGroup:
+  case Help, Logging, Dependency, Scala, Java, Entrypoint, BSP, Markdown, Benchmarking,
+    `Scala Native`, `Build Tool export`, `Scala.js`, Debug, Uninstall, Watch, Run, Config, Update,
```

Comment:
```suggestion
    ScalaNative, BuildToolExport, ScalaJS, Debug, Uninstall, Watch, Run, Config, Update,
```
maybe and have a custom toString? I just prefer to avoid values enclosed in backticks, there're less clear and a lot of the places don't highlight them well (for example here)

Reply:
Should be better now